app.py

```diff
@@ -85,7 +88,7 @@ def schedule_repositories(repositories: Set[str]) -> None:
 
     :param repositories:str: List of repositories in string format: repo1,repo2,...
     """
-    oc = OpenShift()
+    oc = OpenShift(infra_namespace=THOTH_INFRA_NAMESPACE, middletier_namespace=THOTH_MIDDLETIER_NAMESPACE)
```

Comment:
no need for this PR :) these env variables are standardized across all Thoth components and they are taken when you instantiate the Opensfhit class :) Have a look here: https://github.com/thoth-station/common/blob/4152294f4410a943959e9e4e8f903e01d33ee029/thoth/common/openshift.py#L129